docs/atoz.md

```diff
@@ -0,0 +1,487 @@
+
+~~~bash
+[Unit]
+Description=wetty
+After=network.target
+
+[Service]
+Type=simple
+WorkingDirectory=%h
+ExecStart=/bin/bash -c "$$(source /home/$$(whoami)/.nvm/nvm.sh && nvm which 12) /home/$$(whoami)/bin/wetty $$(eval echo $(cat /home/$$(whoami)/.config/wetty/config))"
+TimeoutStopSec=20
+KillMode=mixed
+Restart=always
+RestartSec=2
+
+[Install]
+WantedBy=multi-user.target
+~~~
+
+Press `ctrl` + `x` and then press `y` to save then press `enter` to confirm and exit `nano`.
+
+#### Activating your service
+
+The you can enable and start your service.
+
+~~~bash
+systemctl --user enable --now wetty
+~~~
+
+#### Managing your services
+
+These commands will help you manage your service.
+
+~~~bash
+systemctl --user daemon-reload
+systemctl --user status wetty
+systemctl --user start wetty
+systemctl --user stop wetty
+systemctl --user restart wetty
+systemctl --user disable --now wetty
+systemctl --user enable --now wetty
+~~~
+
+### Nginx reverse proxy
+
+If you want to use nginx as a revers proxy here is the configuration file you can use.
+
+Please modify these specific environment settings:
+
+**Why?** This will disable generic port access to the application and force traffic via the nginx reverse proxy. 
+
+~~~bash
+--host 127.0.0.1
+~~~
+
+**Why?** This change is so that our application does not attempt to load as the web root of `/` for nginx.
+
+~~~bash
+--base /wetty/
+~~~
+
+Now you can use this nginx configuration file.
+
+**Note:** we are using `https` here `https://127.0.0.1:3000/wetty;` because we configured `wetty` to run via `https`
+
+The copy and paste this into the `https` server block of your enable server configuration file.
+
+~~~nginx
+location /wetty {
+    proxy_pass https://127.0.0.1:3000/wetty;
+    #
+    proxy_pass_request_headers on;
+    #
+    proxy_set_header Host $host;
+    #
+    proxy_http_version 1.1;
+    #
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header X-Forwarded-Proto $scheme;
+    proxy_set_header X-Forwarded-Protocol $scheme;
+    proxy_set_header X-Forwarded-Host $http_host;
+    proxy_set_header X-NginX-Proxy true;
+    #
+    proxy_set_header Upgrade $http_upgrade;
+    proxy_set_header Connection $http_connection;
+    proxy_read_timeout 43200000;
+    proxy_set_header X-Forwarded-Ssl on;
+    #
+    proxy_redirect off;
+    proxy_buffering off;
+}
+~~~
+
+Press `ctrl` + `x` and then press `y` to save then press `enter` to confirm and exit `nano`
+
+Now you would need to reload nginx service using this command:
+
+~~~bash
+systemctl restart nginx
+~~~
+
+#### Accessing the web interface via nginx
+
+Visit the URL format `https://YourIP/wetty` and you can access `wetty`. This command will generate the correct URL you need to visit.
+
+~~~bash
+echo https://$(curl -s4 icanhazip.com)/wetty
+~~~
+
+### Configuration  reference
+
+`wetty -h` configuration options for reference.
+
+~~~bash
```

Comment:
can we use ` ``` ` ranther then `~~~`

Reply:
resolved with https://github.com/userdocs/wetty/commit/e5c6e2ce8a0b06f2be359552d4da8298351e530d